drsm/README.md

```diff
@@ -6,22 +6,49 @@ SPDX-License-Identifier: Apache-2.0
 -->
 # Distributed Resource Sharing Module (DRSM)
 
+## Block diagram
+
+![DRSM Usage in AMF](/drsm/images/drsm.png)
+
+## Introduction
+
+It is very common in micro services based architecture to share resources among multiple instances of the microservices.Just to explain the DRSM concepts and design we can take an example of the Access Management Function (AMF) network function. e.g. gNodeB & AMF connect over the N2 interface and NGAP protocol is used for this interface. 3gpp defines the specification of this interface. NGAP uses SCTP transport protocol for communication between gNodeB & AMF. Multiple User Equipment (UE) signaling traffic is sent over the N2 interface. Users are identified by NGAPIDs. Each subscriber is identified by the NGAAP ID of AMF & NGAAP ID of gNodeB. So it's very important for AMF, gNodeB to assign a unique NGAPP ID to UE. When single AMF instance is running in the system then there is no issue and assigned IDs are unique. But when multiple instances of AMF are running in the system then the DRSM module helps the AMF instance for unique ID assignment.
+
+## Module
+
+DRSM is a go module and can be used by go program to start using the resource allocation functionality. In 5G core it is used by SCTP load balancer & AMF.
```

Comment:
I see that this module is only used by the `amf` and `smf`. I do not see that is being used by the `sctplb` module. Or am I misunderstanding something here?

Reply:
SCTPLB Code is with Bilal and should be available soon.

Reply:
Ok, thanks!